PrebidMobile/PrebidMobileRendering/PluginRenderer/PrebidMobilePluginRenderer.swift

```diff
@@ -0,0 +1,48 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+
+@objc public protocol PrebidMobilePluginRenderer: AnyObject {
+    
+    @objc var name: String { get }
+    @objc var version: String { get }
+    @objc var data: [AnyHashable: Any]? { get }
+    
+    var transactionFactory: PBMTransactionFactory? { get set }
```

Comment:
`PBMTransactionFactory` looks like Prebid SDK's internal class. It doesn't looks like third party renderers shoud utilize it somehow. 

Please explain the purpose of this property in the protocol and how it is used in third-party plugins. Or just remove it if it doesn't make sense for plugins.

Reply:
@github-richard-depierre, we should still remove `PBMTransactionFactory` from the public protocol.